### PR TITLE
Stac 5977 local install

### PR DESCRIPTION
**Pull request by Voronenko**
### What does this PR do?

Supports offline installation modes

### CODE REVIEW

File: test/molecule-role/molecule/localinstall/molecule.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    image: ami-0451e9d3427711cb1    # CentOS Linux 6.10 x86_64 HVM EBS ENA 1901_01
+    instance_type: t2.micro
+    vpc_subnet_id: subnet-fa36adb2  # eu-west-1a
+    region: eu-west-1
+    groups:
+      - agent_linux_vm
+    ssh_user: centos
+    user_data: files/agent/none.txt
+
+  - name: receiver
```
Comment (TizianoPerrucci) [inline test/molecule-role/molecule/localinstall/molecule.yml]: remove receiver and its play

File: test/molecule-role/molecule/localinstall/molecule.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+provisioner:
+  name: ansible
+  config_options:
+    defaults:
+      any_errors_fatal: true
+  playbooks:
+    create: create.yml
+    prepare: prepare.yml
+    destroy: destroy.yml
+  inventory:
```
Comment (TizianoPerrucci) [inline test/molecule-role/molecule/localinstall/molecule.yml]: inventory is not needed

File: test/molecule-role/molecule/localinstall/molecule.yml (hunk 1 above)
Comment (Voronenko) [reply test/molecule-role/molecule/localinstall/molecule.yml]: eliminated

File: test/molecule-role/molecule/localinstall/molecule.yml (hunk 2 above)
Comment (Voronenko) [reply test/molecule-role/molecule/localinstall/molecule.yml]: eliminated

**Pull request merged**